server/src/systems/spawn.rs

```diff
@@ -232,6 +223,10 @@ mod test {
         world.register::<BoundingCircle>();
         world.register::<Health>();
         world.register::<Respawn>();
+
+        let resources_path = application_root_dir().unwrap().join("../resources");
```

Comment:
For development it could be OK, but otherwise it weird a bit, that it accesses something out of the application root dir.

Reply:
I totally agree with you but somehow both server and client should access the "common" resources dir. It can not be packed into both of them.

The same "../resources" happens in server/main.rs and in client/main.rs

Reply:
I see. Currently for development it is OK, but for longer term it should be fixed somehow.
It could be a symlink, but it might not work for every platform. Or at install time the ron files could be copied to both server and the client.
And later on, the weapons can be a bit more dynamic than now. For example their parameters can be improved, based on how they were crafted. In this case the server should send the weapon details to the client. That's why we don't have to fix this issue at this moment.